packages/teams-js/src/public/visualMedia.ts

```diff
@@ -0,0 +1,274 @@
+import { sendAndHandleSdkError } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
+import { DevicePermission } from './interfaces';
+import { runtime } from './runtime';
+
+/**
+ * Interact with image and video. It lets the app developer ask the user to get images or video from their camera / camera roll / file system.
+ * @beta
+ */
+export namespace visualMedia {
+  /**
+   * @hidden
+   * Hide from docs
+   * --------
+   * All properties common to Image and Video Props
+   * @beta
+   */
+  interface VisualMediaProps {
+    /**
+     * @hidden
+     * Lets the developer specify the media source
+     */
+    source: Source;
+
+    /**
+     * @hidden
+     * Optional; Specify in which mode the camera will be opened.
+     * Default value is Photo
+     */
+    startMode?: CameraStartMode;
+
+    /**
+     * @hidden
+     * Optional; indicate if user is allowed to move between front and back camera
+     * Default value is true
+     */
+    cameraSwitcher?: boolean;
+    /**
+     * max limit of media allowed to be selected in one go, current max limit is 10 set by lens-sdk.
+     */
+    visualMediaCount: number;
+  }
+  /**
+   *  All properties in ImageProps are optional and have default values in the platform
+   *  Additional properties for image in mobile devices
+   * @beta
+   */
+  export interface ImageProperties extends VisualMediaProps {
+    /**
+     * Optional; indicate if inking on the selected Image is allowed or not
+     * Default value is true
+     */
+    ink?: boolean;
+
+    /**
+     * Optional; indicate if putting text stickers on the selected Image is allowed or not
+     * Default value is true
+     */
+    textSticker?: boolean;
+
+    /**
+     * Optional; indicate if image filtering mode is enabled on the selected image
+     * Default value is false
+     */
+    enableFilter?: boolean;
+
+    /**
+     * Optional; Lets the developer specify the image output formats, more than one can be specified.
+     * Default value is Image.
+     */
+    imageOutputFormats?: ImageOutputFormats[];
```

Comment:
>imageOutputFormats [](http://example.com/codeflow?start=4&length=18)

Is this letting the developer pick which kinds of files are allowed to be selected? Or is it letting the developer force the files to be converted to these formats?

Reply:
Should be the former one: letting the developer specify which kinds of files are allowed to be selected. Updated the field naming to `imageFormatsAllowedToBeSelected`